README.md

```diff
@@ -261,6 +261,36 @@ ENV BROKER_TOKEN      secret-broker-token
 ENV ARTIFACTORY_URL   <yourdomain>.artifactory.com
 ```
 
+### Nexus
```

Comment:
How did you approach the Artifactory integration - since this is a public repo and we are working on Nexus, we shouldn't put this in the README just yet? Any other considerations?

Reply:
will remove